bears/gherkin/GherkinLintBear.py

```diff
@@ -0,0 +1,133 @@
+import json
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from coalib.results.Result import Result
+
+_setting_map = {True: 'on',
+                False: 'off'}
+
+
+@linter(executable='gherkin-lint',
+        use_stderr=True,
+        use_stdout=False,
+        global_bear=True)
+class GherkinLintBear:
+    """
+    Use Gherkin to run linting on feature files
+    """
+
+    LANGUAGES = {'Gherkin'}
+    REQUIREMENTS = {NpmRequirement('gherkin-lint', '2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala_devel@googlegroups.com'}
+    LICENSE = 'AGPL_3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
+
+    def process_output(self, output, filename, file):
+        stderr = json.loads(output)
+        for fileErr in stderr:
+            filePath = fileErr['filePath']
+            for err in fileErr['errors']:
+                yield Result.from_values(
+                    origin='{} ({})'.format(
+                        self.__class__.__name__,
+                        err['rule']),
+                    message=err['message'],
+                    line=int(err['line']),
+                    file=filePath)
+
+    @staticmethod
+    def generate_config(filename, file,
+                        indentation: bool = False,
```

Comment:
please use the standard setting name for this.

Reply:
Sorry, but I couldn't find the standard setting name... i found `indent_size` but that specifies the size of the indent. Should it be `allow_indent`?

Reply:
@anctartica , does it have a fixed indentation size?

If so, use `indent_size = <x>` and allow only the fixed indentation size or `0`

Reply:
I dont see any of the test files which dont use indentation.

Reply:
`indentation` has to be configured like this 
`"indentation" : [
    "on", {
      "Feature": 0,
      "Background": 0,
      "Scenario": 0,
      "Step": 2,
      "Examples": 0,
      "example": 2,
      "given": 2,
      "when": 2,
      "then": 2,
      "and": 2,
      "but": 2,
      "feature tag": 0,
      "scenario tag": 0
    }
  ]`

Reply:
should it be removed from here? @jayvdb

Reply:
yup ; remove options that we cant easily support.

they can be added later.